esp/services/ws_sql/ws_sqlService.cpp

```diff
@@ -831,8 +831,7 @@ void CwssqlEx::processMultipleClusterOption(StringArray & clusters, const char
         hashoptions.appendf("\n#OPTION('AllowedClusters', '%s", targetcluster);
         ForEachItemIn(i,clusters)
         {
-            if (!isValidCluster(clusters.item(i)))
-                throw MakeStringException(-1, "Invalid alternate cluster name: %s", clusters.item(i));
+            validateTargetName(clusters.item(i));
```

Comment:
Did we say that the check and the throw should be moved inside the API call (here validateTargetName()) so that every caller  does not have to check and throw?

Reply:
possibly. it's fine as is.